pkg/testcommon/testcommon.go

```diff
@@ -41,7 +40,7 @@ func (site *TestSite) Prepare() error {
 	site.Dir = testDir
 	log.Debugf("Prepping test for %s.\n", site.Name)
 	err := os.Setenv("DRUD_NONINTERACTIVE", "true")
-	util.CheckErr(err)
+	checkErr(err)
```

Comment:
Consider using the "black box" technique as in https://github.com/drud/ddev/pull/193 - or alternately, just `if err { fatalf } ` here.